### DISCUSSION TITLE
Projektarbeit

### DISCUSSION THREAD

**Discussion by Wildeswiesel** [Wildeswiesel/Space-Waffel#8] [General]
am 22.01.2024 wurde uns die Aufgabe gestellt Gruppen zu finden und uns ein Thema für eine Projektarbeit zu überlegen 

Meine Gruppe hat sich überlegt eine To-Do Liste zu machen welche von einer Website aus sichtbar ist (derzeit ist die überlegung Flask zu nehmen) & ein paar kleine funktionen hat wie z.b. eine Suchfunktion

gegen Ende sollten wir noch unsere learning outcomes definieren -> dort hatten wir fesgelegt arbeiten mit datenbanken & Websiten in bezug auf python

---

Die & letzte Woche hatten ich tendenziell eher weniger Zeit, da wir viele abgaben in theoretischer Informatik hatten und somit eher weniger programmiert habe. Desweitern hatten wir auch noch die Abgabe in Betriebswirtschaftslehre die am Sonntag fertig sein sollte.
Das einzige wo ich kurz was gemacht habe war meinen Arduino ins Wlan mit ein zu binden (code wird noch hoch geladen)

**Comment by d4c4n** [comment]
Ich fühle den Aspekt mit der fehlenden Zeit sehr. Andere Dinge haben einfach mehr Raum eingenommen. Wird vielleicht auch wieder besser :)

**Comment by stechmo** [comment]
Leider ging es mir diese Woche auch so, ich weiß nicht was ich zuerst tun soll, damit alles rechtzeitig fertig wird..

**Comment by SvenSrc** [comment]
Vermutlich ging es allen so mit der Zeit, dass es zu viel wurde diese Woche
